playbook_ui.gemspec

```diff
@@ -35,6 +35,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency "rainbow", "2.2.2"
   s.add_development_dependency "rspec", "~> 3.8.0"
   s.add_development_dependency "rubocop", "0.72.0"
+  s.add_development_dependency "simple_form"
```

Comment:
Tie to a version?

Reply:
@wadewinningham I can specify a more specific version.

Reply:
I wouldn't tie to a single version, but something more flexible like a trusted range. I say that because playbook is a library, and tying to a single version can be too hard.